app/models/rubygem.rb

```diff
@@ -82,6 +82,23 @@ def self.monthly_short_dates
     monthly_dates.map { |date| date.strftime("%m/%d") }
   end
 
+  def self.most_downloaded_by_db(limit=10)
```

Comment:
@sferik stats controller have ```Rubygem.downloaded(10)``` and this is the only place that calls Rubygem.downloaded. So we can just use limit parameter as 10 but you're right I should also update downloaded method limit parameter with 10. Sounds good?